ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -593,8 +593,8 @@ public boolean hasSupportForMessage(final int messageCode) {
   @Override
   public String toString() {
     return String.format(
-        "PeerId %s, reputation %s, validated? %s, disconnected? %s, client: %s, connection %s, enode %s",
-        nodeId(),
+        "PeerId: %s, %s, validated? %s, disconnected? %s, client: %s, %s, %s",
+        getId().slice(0, 16),
```

Comment:
If this is a truncated value maybe should indicate that, e.g. with an `...`?

Reply:
done